manifests/00-crd-consoleexternalloglinks.yaml

```diff
@@ -0,0 +1,85 @@
+  annotations:
+    displayName: ConsoleExternalLogLinks
+    description: ConsoleExternalLogLink is an extension for customizing OpenShift
+      web console log links.
+spec:
+  scope: Cluster
+  group: console.openshift.io
+  versions:
+    - name: v1
+      served: true
+      storage: true
+  names:
+    plural: consoleexternalloglinks
+    singular: consoleexternalloglink
+    kind: ConsoleExternalLogLink
+    listKind: ConsoleExternalLogLinkList
+  additionalPrinterColumns:
+    - name: Text
+      type: string
+      JSONPath: .spec.text
+    - name: HrefTemplate
+      type: string
+      JSONPath: .spec.hrefTemplate
+    - name: Age
+      type: date
+      JSONPath: .metadata.creationTimestamp
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      description: ConsoleExternalLogLink is an extension for customizing OpenShift
+        web console log links.
+      type: object
+      required:
+        - spec
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          description: Standard object's metadata.
+          type: object
+        spec:
+          description: ConsoleExternalLogLinkSpec is the desired log link configuration.
+            The log link will appear on the logs tab of the pod details page.
+          type: object
+          required:
+            - hrefTemplate
+            - text
+          properties:
+            hrefTemplate:
+              description: "hrefTemplate is an absolute secure URL (must use https)
+                for the log link including variables to be replaced. Variables are
+                specified in the URL with the format ${variableName}, for instance,
+                ${containerName} and will be replaced with the corresponding values
+                from the resource. Resource is a pod. Supported variables are: - ${resourceName}
+                - name of the resource which containes the logs - ${resourceUID} -
+                UID of the resource which contains the logs               - e.g. `11111111-2222-3333-4444-555555555555`
+                - ${containerName} - name of the resource's container that contains
+                the logs - ${resourceNamespace} - namespace of the resource that contains
```

Comment:
This is missing `resourceNamespaceUID` from https://github.com/openshift/api/pull/466

It would be good to fix the formatting here

Reply:
Yikes the `generate` definitely did not format this well.

Reply:
Bah, and we have a `/hold` on the updates with the `patterns`, so I'll have to reconcile by hand.  😢  
(https://github.com/openshift/cluster-config-operator/pull/101/files)

Reply:
Updated.